src/main/java/pl/touk/sputnik/processor/ktlint/KtlintProcessor.java

```diff
@@ -0,0 +1,83 @@
+package pl.touk.sputnik.processor.ktlint;
+
+import com.github.shyiko.ktlint.core.KtLint;
+import com.github.shyiko.ktlint.core.RuleSet;
+import com.github.shyiko.ktlint.core.RuleSetProvider;
+import org.apache.commons.io.IOUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import pl.touk.sputnik.configuration.Configuration;
+import pl.touk.sputnik.configuration.GeneralOption;
+import pl.touk.sputnik.review.Review;
+import pl.touk.sputnik.review.ReviewProcessor;
+import pl.touk.sputnik.review.ReviewResult;
+import pl.touk.sputnik.review.filter.KotlinFilter;
+import pl.touk.sputnik.review.transformer.FileNameTransformer;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ServiceLoader;
+
+class KtlintProcessor implements ReviewProcessor {
+    private static final String SOURCE_NAME = "ktlint";
+
+    private final List<RuleSet> ruleSets;
+    private final List<String> excludedRules;
+
+    KtlintProcessor(Configuration configuration) {
+        excludedRules = parseExcludedRules(configuration);
+        ruleSets = findRuleSets();
+    }
+
+    private static List<RuleSet> findRuleSets() {
```

Comment:
Why static?

Reply:
Because it can be static - these methods do not use fields so why not:) I will change it to keep convention...